.github/workflows/Main.yml

```diff
@@ -193,7 +193,7 @@ jobs:
 
     - name: Test
       shell: bash
-      run: build/reldebug/test/unittest --force-reload --force-storage
+      run: build/reldebug/test/unittest --force-reload --force-storage --summarize-failures
```

Comment:
Instead of making this a command line option maybe we can make it an environment variable? That way this also works with the CI runs that run e.g. `make allunit`.